docs/data/base/components/modal/modal.md

```diff
@@ -0,0 +1,127 @@
+---
+product: base
+title: React Modal component
+components: ModalUnstyled
+githubLabel: 'component: modal'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#dialog_modal
```

Comment:
This is required to make the ComponentLinkHeader's bundle size link point to the right location:

```suggestion
waiAria: https://www.w3.org/TR/wai-aria-practices/#dialog_modal
packageName: '@mui/base'
```